test-framework/src/main/java/api/app/astrodao/com/core/enums/HttpStatus.java

```diff
@@ -0,0 +1,599 @@
+package api.app.astrodao.com.core.enums;
+
+import org.springframework.lang.Nullable;
+
+public enum HttpStatus {
```

Comment:
Use inner Java class HttpURLConnection with implemented HTTP statuses. Import class instead of implement your own class